From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Robbie Harwood <rharwood@redhat.com>
Date: Thu, 7 Jul 2022 16:56:41 -0400
Subject: [PATCH] Fix mandoc invocation to not produce garbage

Bizarrely, mandoc doesn't default to outputting man - the default is
"locale", which is either ASCII or UTF-8 (by locale).  This output is
supposed to be some kind of plain-text, but it's formatted so strangely
I'm not sure what the purpose is.  Regardless, it doesn't go well to
feed this into man(1).

Tell mandoc explicitly to produce man pages.

Signed-off-by: Robbie Harwood <rharwood@redhat.com>
---
 Make.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

NOTE: This change is already present in pesign 116. The patch body is a no-op so the packaging series continues to apply cleanly after the version bump.

diff --git a/TODO b/TODO
--- a/TODO
+++ b/TODO
@@ -1 +1 @@
-Stuff that needs to get done:
+Stuff that needs to get done:
